libs/shared/feature/form-nuv/src/lib/components/steps/formly/formly-steps.component.html

```diff
@@ -42,10 +43,22 @@ <h1>{{ messages.confirmationStepLabel }}</h1>
     </ng-template>
   </nuverial-stepper>
 
-  <section class="footer-actions" *ngIf="field.fieldGroup">
-    <nuverial-footer-actions [buttonsDisabled]="updatingTransaction" [footerActions]="actions" (actionSelected)="onActionClick($event)">
-    </nuverial-footer-actions>
-  </section>
+  <ng-container [ngSwitch]="formWorkflow">
+    <ng-container *ngSwitchCase="formStateWorkflowMode.AdminBuilder" [ngTemplateOutlet]="adminBuilder"></ng-container>
+    <ng-container *ngSwitchCase="formStateWorkflowMode.PublicIntake" [ngTemplateOutlet]="publicIntake"></ng-container>
+    <ng-container *ngSwitchDefault>{{stepperMode}} mode not implemented</ng-container>
```

Comment:
I'm not sure if we actually need the default case here but this would be a better message
```suggestion
    <ng-container *ngSwitchDefault>Validation/Save logic not yet implemented for this workflow</ng-container>
```